README.md

```diff
@@ -1,2 +1,267 @@
-# Microverse_code-review
-Learning how to review code, a project from the Microverse curriculum
+<a name="readme-top"></a>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Hello microverse details
+
+- [📖 About the Project ](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HELLO-MICROVERSE] <a name="about-project"></a>
```

Comment:
- [ ] Kindly be more polite and professional, it seems like you commanding through your comments. Please be more respectful in your comments which will help the student to feel comfortable while receiving the comment.

### Your Comment
> Remove the square brackets as they only served to indicate where to input your text.